File: thesis/src/code/evaluator.py
from evaluation_utils import EvaluationResult, Message

def evaluate(expected, actual, arguments):
    # Controleer of de gekregen waarde overeenkomt.
    correct = expected == actual
    messages = []
    if not correct:
        messages.append(Message("Hallo, dit is niet juist!"))
    # We geven geen verwachte waarde mee; TESTed neemt de waarde uit het testplan.
    return EvaluationResult(correct, None, actual, messages)

File: tests/exercises/partial-function/evaluation/Evaluator.cs
using Tested;
using System.Collections.Generic;

public class Evaluator
{
    public static EvaluationResult Evaluate(object actual)
    {
        string shown = actual != null ? actual.ToString() : "";
        return new EvaluationResult(true, shown, shown, new List<Message>());
    }
}
